src/20240718_AaveV3Ethereum_EventsGrant2024/EventsGrant2024.md

```diff
@@ -0,0 +1,44 @@
+
+1. Hackathons & Bounties: We want to encourage developers to build on Aave Protocol, as well as GHO. These hackathons not only provide a platform for the Aave community to connect with talented developers in both web3 and web2, and to remain visible and competitive in a growing industry, but also lead to the creation of innovative DeFi projects that enhance the Aave ecosystem. We propose to sponsor one ETHGlobal hackathon in Q4 in Bangkok.
+
+2. Open Finance Day: We are hosting Open Finance Day in partnership with key players in DeFi for high quality talks, networking, and light bites. Talks will include members of the DeFi community, Aave Labs, and Aave DAO community members and service providers.
+
+3. Side Events: Hosting side events alongside major conferences is an opportunity to increase community awareness, foster connections, and share technical knowledge, and industry insights. Typically featuring panels, workshops, and discussions led by industry experts, side events are designed to strengthen awareness of Aave and GHO technology among strategic audiences, reach new target audiences, and ensure Aave and GHO remain top-of-mind within and outside the community.
+
+4. Merch: Aave Labs seeks to create highly coveted Aave and GHO branded merchandise to distribute at events, as part of community engagement and awareness-building efforts as well as to proliferate the new visual identity and the beloved Ronnie ghost. Aave Labs will continue to push for the use of sustainable materials and creation of merchandise people actually wear, use, and love.
+
+5. rAAVE: To celebrate the Aave community’s welcoming culture, we aim to host two more editions of the flagship rAAVE event. Throughout the past years rAAVE has positioned itself to be the most sought after event in the ecosystem. They foster community spirit and create memorable experiences. We will integrate the GHO Pass for ticketing logistics and manage press relations and social media to maximize awareness and inclusion. This year’s events will be hosted at EthCC in Brussels and Devcon in Bangkok. To mitigate some costs for rAAVE, we are planning to potentially obtain co-sponsorship with current or potentially new partners for the Aave ecosystem. Due to the event’s high visibility, it is crucial that if co-sponsorship is obtained, the selected partners are aligned with the Aave Community and this is conducted in a balanced manner.
+
+After covering the aforementioned main sponsorships, any remaining budget will be directed towards side events related to the security of DeFi and smart contract technology. Any excess funds from this grant will be rolled over to be used for events during Q1 2025. A recap on last year’s events was posted to governance last December, here.
+Aave Labs shall cover all expenses for its team members, and this proposal does not request any funds for these expenses, or include any compensation for Aave Labs’ work.
+
```

Comment:
Missing here, to which address this is sent, and also that the amount is transferred from the collector